api/services/ocean/commons/paths/extendedResourceDefinition.yaml

```diff
@@ -45,8 +45,8 @@ delete:
   tags:
     - "Ocean AWS"
   parameters:
-      - $ref: "../../../../commons/parameters/accountId.yaml"
-      - $ref: "../../commons/parameters/oceanClusterId.yaml"
+    - $ref: "../../../../commons/parameters/accountId.yaml"
+    - $ref: "../../commons/parameters/oceanExtendedResourceDefinitionId.yaml"
```

Comment:
This is more than a syntactic change, did you check with our API source-code that these are the correct parameters?

Reply:
Yes

https://github.com/spotinst/spotinst-mcs-core/blob/8b65dbf5c3b7b0003d109fe07022906e2e655ddc/src/main/java/com/spotinst/mcs/core/api/resources/app/OceanResource.java